setup.py

```diff
@@ -20,7 +20,7 @@
     license=license,
     packages=find_packages(exclude=('tests', 'docs')),
     install_requires=[
-        'ethereum == 1.3.7',
+        'ethereum==2.0.4',
         'serpent',
```

Comment:
We could to implement [Semantic Version](http://semver.org/) and freeze the dependencies version with every Viper version.